backend/core/src/auth/guards/local-auth.guard.ts

```diff
@@ -1,5 +0,0 @@
-import { Injectable } from "@nestjs/common"
```

Comment:
Note to self: Check if MFA is wanted on Detroit and if not ensuring it is set as false on user create